packages/utilities/src/utilities.client.ts

```diff
@@ -158,18 +158,25 @@ export function normalizeUrl(url: string, keepFragment?: boolean) {
 }
 
 // Helper function for markdown rendered marked
-// Renders links outside apify.com in readme with rel="noopener noreferrer nofollow" and target="_blank" attributes
-export function markedSetNofollowLinks(href: string, title: string, text: string) {
+// If passed hostname, it renders links outside that hostname in readme with rel="noopener noreferrer" and target="_blank" attributes
+// And links outside apify.com in readme with rel="noopener noreferrer nofollow" and target="_blank" attributes
+export function markedSetNofollowLinks(href: string, title: string, text: string, hostname?: string) {
     let urlParsed: URL;
     try {
         urlParsed = new URL(href);
     } catch (e) {
         // Probably invalid url, go on
     }
     const isApifyLink = (urlParsed! && /(\.|^)apify\.com$/i.test(urlParsed.hostname));
-    return (isApifyLink)
-        ? `<a href="${href}">${title || text}</a>`
-        : `<a rel="noopener noreferrer nofollow" target="_blank" href="${href}">${title || text}</a>`;
+    const isSameHostname = !hostname || (urlParsed! && urlParsed.hostname === hostname);
+
+    if (isApifyLink && isSameHostname) {
+        return `<a href="${href}">${title || text}</a>`;
```

Comment:
Out of curiosity - why don't we use the DOM API (`createElement`, `setAttribute` etc)? I'm sure this is handled in many other layers, but syntax like this always makes me think of XSS. I see the file is called `utilities.client.ts`, this means it only runs in browsers, right? (so DOM API should be always available)

Reply:
I was thinking the same at first and maybe loading the current host name just inside the function, but even it's called `utilities.client.ts` it is used in daemons and console backend 🤔

Reply:
Gotcha — weird indeed. Since this has worked until now and nobody complained, we don't need to rethink it right now. thanks!